lib/plugins/labels.js

```diff
@@ -19,19 +19,19 @@ module.exports = class Labels extends Diffable {
   }
 
   find () {
-    return this.github.issues.getLabels(this.wrapAttrs({})).then(res => res.data)
+    return this.github.issues.listLabelsForRepo(this.wrapAttrs({})).then(res => res.data)
```

Comment:
Is it possible to have more than 100 labels, so that pagination is required?

Reply:
Good point. Yes, the code would look like this:

```js
const options = this.github.issues.listLabelsForRepo.endpoint.merge(this.wrapAttrs({per_page: 100}));
return this.github.paginate(options)
```

` this.github.paginate(options)` already returns an array, so no need to map to `res.data`